src/test/java/pages/commonPage.java

```diff
@@ -70,11 +70,27 @@ public void choosePrisoners() {
     public void chooseAnotherPrisoner() {
         click("xpath","//*[@id='visitor-4729773']");
     }
+
+    public void chooseVisitorFromBristol(){
+        click("xpath","//*[@id='visitor-4729777']");
+    }
+
+    public void choosePrisonerB() {
+        click("xpath","//*[@id='visitor-4729791']");
+
+    }
     public void clickOnSignOut() {
         click("xpath","/html/body/header/div/nav/ul/li[2]/a");
     }
     public void errorMsgDisplayed(String errorMsg) {
         isElementDisplayed("xpath","//*[@id='error-detail']");
     }
 
+    public void chooseBristol(){
+        click("xpath","//a[contains(@data-test, 'change-establishment')]");
+        selectRadioButton("xpath","//*[@id='BLI']");
+
+    }
+
+
 }
```

Comment:
blank line at EOF and a few other files below

Reply:
address this